conf/relco/jira_recurrent_tickets.json

```
@@ -57,6 +57,48 @@
             "component": "Relco tasks",
             "description": "https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Static+documentation+update#Staticdocumentationupdate-Schemadocumentation",
             "summary": "Update Compara schema documentation in Ensembl plugins"
+         },
+         {
+            "assignee": "<RelCo>",
+            "component": "Relco tasks",
+            "description": "https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Flat+file+dumps#Flatfiledumps-Updatingdatadumpregistryfiles",
+            "summary": "Update Compara data dump registry files"
+         },
+         {
+            "assignee": "<RelCo>",
+            "component": "Relco tasks",
+            "description": "https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Flat+file+dumps#Flatfiledumps-Trackingdatadumpserverusage",
+            "summary": "Fungi data dump server usage"
```

```suggestion
            "summary": "Fungi data dump server usage",
            "server": "mysql-ens-compara-prod-4"
```
I put this suggestion here with the idea discussed that Production could incrementally copy the required tables back to the existing release databases on our servers - in which case the server will be set. This would also follow our usage for the next couple of releases, while Production team are understaffed, and we will need to run the FTP dumps. ourselves. Not sure if this will work syntactically with the Jira API - but it's an idea.

---

Setting the `server` field like this is not currently supported by `create_compara_release_JIRA_tickets.pl`. But even if it we could, I feel we need to ask if we should. Wouldn't this cause the data-dump tickets to be displayed on the Compara server usage dashboard from Compara branching onwards? I'm not sure it would be useful to display them so early.

---

Not unless the ticket was put in progress from branching would the server usage show the server as being used for the ticket at the time